packages/remix-cloudflare/package.json

```diff
@@ -16,9 +16,9 @@
     "@remix-run/server-runtime": "1.3.3"
   },
   "peerDependencies": {
-    "@cloudflare/workers-types": "^2.2.2"
+    "@cloudflare/workers-types": "^3.4.0"
```

Comment:
This would be a breaking change

Reply:
Hmmm, I guess so. I can look into allowing more versions in this range for the peerdependencies.